worlds/shivers/__init__.py

```diff
@@ -22,6 +22,8 @@ class ShiversWorld(World):
     Shivers is a horror themed point and click adventure. Explore the mysteries of Windlenot's Museum of the Strange and Unusual.
     """
 
+    pot_completed_list: list[...]
+
```

Comment:
Godl is on vacation for 3 weeks, but this would need to get fixed.

```suggestion
    pot_completed_list: List[...]
```

Reply:
Fixed